deepchecks/nlp/checks/data_integrity/property_label_correlation.py

```diff
@@ -0,0 +1,139 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""The feature label correlation check module."""
+import typing as t
+
+import pandas as pd
+
+import deepchecks.ppscore as pps
+from deepchecks.core import CheckResult, ConditionCategory, ConditionResult
+from deepchecks.core.check_utils.feature_label_correlation_utils import get_pps_figure, pd_series_to_trace
+from deepchecks.nlp import Context, SingleDatasetCheck
+from deepchecks.tabular.utils.messages import get_condition_passed_message
+from deepchecks.utils.strings import format_number
+from deepchecks.utils.typing import Hashable
+
+__all__ = ['PropertyLabelCorrelation']
+
+PLC = t.TypeVar('PLC', bound='PropertyLabelCorrelation')
+
+pps_url = 'https://docs.deepchecks.com/en/stable/checks_gallery/tabular/' \
+          'train_test_validation/plot_feature_label_correlation_change.html'
+pps_html = f'<a href={pps_url} target="_blank">Predictive Power Score</a>'
+
+
+class PropertyLabelCorrelation(SingleDatasetCheck):
+    """Return the PPS (Predictive Power Score) of all features in relation to the label.
```

Comment:
features? This docstring should be updated for nlp

Reply:
Done